documentation/meta/decision_making/additional_practices.md

```diff
@@ -13,10 +13,10 @@ should include the following text:
 ```md
 This discussion is following the Openverse decision-making process. Information
 about this process can be found
-[on the Openverse documentation site](https://docs.openverse.org/reference/decision_making/index.html).
-Requested reviewers or participants will be following this process. If you are
-being asked to give input on a specific detail, you do not need to familiarise
-yourself with the process and follow it.
+[on the Openverse documentation site](/meta/decision_making/index.md). Requested
```

Comment:
Should this link to `docs.openverse.org`, since the actual PR template it's quoting does?

Reply:
Oh that's a good question! This link _should_ get rendered I think 😮 Let's check with the preview before this gets merged!

Reply:
![image](https://github.com/WordPress/openverse/assets/10214785/17fd18f1-3554-43ac-b3ea-f3510611bb87)

Crap, it doesn't :/ I'll revert this